src/oc-id/spec/controllers/profiles_controller_spec.rb

```diff
@@ -63,7 +63,8 @@
     let(:put_user) do
       {
         first_name: 'sally',
-        last_name: 'solly'
+        last_name: 'solly',
+        email: "solly@example.com"
```

Comment:
I think I'm missing something - shouldn't the "should not update user's email" test be failing now?

Reply:
@marcparadise yeah I agree, this change is not required now.